src/output_gstreamer.c

```diff
@@ -555,12 +564,23 @@ static int output_gstreamer_init(void)
 			g_object_set (G_OBJECT (player_), "audio-sink", sink, NULL);
 		}
 	}
-	if (videosink != NULL) {
+	if (video_sink != NULL) {
 		GstElement *sink = NULL;
-		Log_info("gstreamer", "Setting video sink to %s", videosink);
-		sink = gst_element_factory_make (videosink, "sink");
+		Log_info("gstreamer", "Setting video sink to %s", video_sink);
+		sink = gst_element_factory_make (video_sink, "sink");
 		g_object_set (G_OBJECT (player_), "video-sink", sink, NULL);
 	}
+    if(video_pipe != NULL) {
```

Comment:
The indentation looks off in this block. Note that this file uses TAB indentation.